.babelrc

```
@@ -0,0 +1,6 @@
+{
+  "plugins": [
+    "@babel/plugin-proposal-class-properties",
+    ["@babel/plugin-proposal-decorators", { "decoratorsBeforeExport": true }]
+  ]
+}
```

Files should end in newline.

```suggestion
}

```